lavamoat/browserify/beta/policy.json

```diff
@@ -3285,17 +3285,6 @@
         "browserify>process": true
       }
     },
-    "@sentry/integrations": {
```

Comment:
These policy changes were caused by the Sentry constants refactor. Previously these were included in the main application bundle solely because the contents were imported from `setupSentry.js`, which imports this package. But now that the constants are imported from the constants file instead, `setupSentry.js` isn't part of the main bundle anymore, so neither is this.